assets/sass/forms.scss

```diff
@@ -64,17 +61,12 @@ form#search {
 // Breakpoint ----------------
 @media (max-width: $default) {
   form#search{
-    @include center-transformX;
```

Comment:
I have no idea what this was supposed to be doing.

Reply:
I fear that's the one that (improperly?) moves the search bar to the left: https://github.com/dscho/git-scm.com/blob/49e42fd582e9eec05aacecd57116b68696f0ec98/assets/sass/mixins.scss#L63-L67

It was [added](https://github.com/dscho/git-scm.com/commit/2445646d0715d3716bd51633af7ac33ad0c1f493#diff-fe68fb1fc606c03aed73a17bc4c6be72d3aca45ab06a31124a378dbc4c855657R63-R66) as part of https://github.com/git/git-scm.com/pull/1644.

Maybe we just delete it?